packages/dcos-checks/buildinfo.json

```diff
@@ -0,0 +1,5 @@
+{
+	"state_directory": true,
+	"username": "dcos_3dt",
```

Comment:
The reason we had to use 3dt username is because dcos-checks will be reusing 3dt system account in EE DC/OS. Also, we are going to rename 3dt to dcos-diagnostics.